README.md

```diff
@@ -1,2 +1,267 @@
+- Bring js-ipfs and go-ipfs to v1.0.0
+
+You can contribute to these Epic by testing, debugging or even coding to either Go or JS IPFS. Both teams have a chat every week that you can join. [Go](https://github.com/ipfs/team-mgmt/blob/master/MGMT_GOLANG_CORE_DEV.md) & [JS](https://github.com/ipfs/team-mgmt/blob/master/MGMT_JS_CORE_DEV.md#weekly-core-dev-team-calls).
+
+**2. Support Software Package Managers into entering the Distributed Web**
+
+Software Package Managers are one of the best systems to take advantage of Content Addressing. Developers and Deployment systems hit Package Managers Registries everyday to download the same bits of code, creating a huge overhead in terms of bandwidth. In addition today, it is incredibly important that we ensure that the code that we run is the code that the author's intended and Content Addressing can help with this goal by adding a Integrity verification step.
+- Kick-off a Package Managers Working Group to work closely with different Package Managers and understand each one challenges and needs.
+- Publish a report on the results from adding the NPM Registry to IPFS.
+- Meet the scalability and performance needs to serve the Package Managers users.
+
+We will booting the Package Managers Working Group soon, meanwhile, check out [NPM on IPFS](https://github.com/ipfs-shipyard/npm-on-ipfs) & [GX](https://github.com/whyrusleeping/gx) to get a feel for what IPFS can contribute to Package Management.
+
+**3. Scale the IPFS Network**
+
+To support the continued growth of the IPFS Network, we plan to optimize our scalability and testing.
+- Deploy a P2P testbed for Distributed Protocols that enables us to spin up hundreds of thousands of nodes on demand and run tests on them.
+- IPFS has an opt-in Telemetry system so that tests can collect key data to understand the experiments.
+
+Our friends from libp2p will be playing the larger role into achieving this goal. If you are a Networks specialist and/or if you like scaling systems to serve hundreds of millions, join the conversation at [Bi-Weekly Call](https://github.com/libp2p/team-mgmt#bi-weekly-call) and learn more.
+
+**4. Create a platform that untangles the mysteries of developing a DApp**
+
+Support the explosion of the DApp ecosystem with underlying tooling, best practices, and community support.
+- Fully launch Peer Star (Peer Base) toolkit to support building Dapps that can run Online, Offline, and in Disjoint Networks.
+- Translate the Identity Manager RFC into code, a sandboxed way for Users to manage their DWeb Identity.
+- Support DApp security needs (API security, multi-tenancy, and secure broadcast updates).
+- Create a guidebook/spec to popularize best practices to deal with unique UX challenges of the DWeb.
+- Continue to contribute to the adoption of the DWeb in the WebPlatform (Brave, libdweb, IPFS companion), making it more accessible to users and Web Developers.
```

Comment:
Is "WebPlatform" meant to be one word?

...I didn't realise that was a thing...I have no idea what I'm doing dog <--me